Formula/chamber.rb

```diff
@@ -0,0 +1,31 @@
+class Chamber < Formula
+  desc "CLI for managing secrets through AWS SSM Parameter Store"
+  homepage "https://github.com/segmentio/chamber"
+  url "https://github.com/segmentio/chamber.git",
```

Comment:
Use the release tarball instead: https://github.com/segmentio/chamber/archive/v1.9.0.tar.gz